bin/sl-run.js

```diff
@@ -24,6 +24,9 @@ var agentOptions = {
   logger: config.logger,
   strongTracer: tracer(),
 };
+if (process.env.SPEED_UP_INTERVAL_FOR_TESTS === 'yes') {
```

Comment:
strong-agent already supports an interval control interface, I just consider this a strong-supervisor specific convenience wrapper for it. I don't want to invent another cross-module environment variable to replace the previous one and just repeat this process in 6 months.